internal/store/store.go

```diff
@@ -0,0 +1,52 @@
+package store
+
+import (
+	"context"
+
+	"cloud.google.com/go/firestore"
+	"github.com/covid19cz/erouska-backend/internal/constants"
+	"github.com/covid19cz/erouska-backend/internal/firebase"
+)
+
+// Storer is a storage abstraction layer interface
+type Storer interface {
+	Doc(string) *firestore.DocumentRef
+	RunTransaction(context.Context, func(context.Context, *firestore.Transaction) error, ...firestore.TransactionOption) error
+}
+
+// Client to interact with storage API
+type Client struct{}
+
+// Doc returns a DocumentRef that refers to the document in the collection with the given identifier.
+func (i Client) Doc(path string) *firestore.DocumentRef {
+	client := firebase.FirestoreClient
+	return client.Collection(constants.CollectionRegistrations).Doc(path)
```

Comment:
The collection name has to be parametrized.

Reply:
do we need it now?
would store.Doc(collectionName, path) work?

Reply:
We do - there are many PRs incoming and they use some other collections.  
`store.Doc(collectionName, path)` is fine :+1:

Reply:
ahh, I see that you are already using multiple collection, ok:

Reply:
solved